ra/ra_test.go

```diff
@@ -284,31 +284,31 @@ func TestValidateContacts(t *testing.T) {
 	_, _, ra, _, cleanUp := initAuthorities(t)
 	defer cleanUp()
 
-	ansible, _ := core.ParseAcmeURL("ansible:earth.sol.milkyway.laniakea/letsencrypt")
-	validEmail, _ := core.ParseAcmeURL("mailto:admin@email.com")
-	otherValidEmail, _ := core.ParseAcmeURL("mailto:other-admin@email.com")
-	malformedEmail, _ := core.ParseAcmeURL("mailto:admin.com")
-	nonASCII, _ := core.ParseAcmeURL("mailto:señor@email.com")
+	ansible := "ansible:earth.sol.milkyway.laniakea/letsencrypt"
+	validEmail := "mailto:admin@email.com"
+	otherValidEmail := "mailto:other-admin@email.com"
+	malformedEmail := "mailto:admin.com"
+	nonASCII := "mailto:señor@email.com"
 
-	err := ra.validateContacts(context.Background(), &[]*core.AcmeURL{})
+	err := ra.validateContacts(context.Background(), &[]string{})
 	test.AssertNotError(t, err, "No Contacts")
 
-	err = ra.validateContacts(context.Background(), &[]*core.AcmeURL{validEmail, otherValidEmail})
+	err = ra.validateContacts(context.Background(), &[]string{validEmail, otherValidEmail})
 	test.AssertError(t, err, "Too Many Contacts")
 
-	err = ra.validateContacts(context.Background(), &[]*core.AcmeURL{validEmail})
+	err = ra.validateContacts(context.Background(), &[]string{validEmail})
 	test.AssertNotError(t, err, "Valid Email")
 
-	err = ra.validateContacts(context.Background(), &[]*core.AcmeURL{malformedEmail})
+	err = ra.validateContacts(context.Background(), &[]string{malformedEmail})
 	test.AssertError(t, err, "Malformed Email")
 
-	err = ra.validateContacts(context.Background(), &[]*core.AcmeURL{ansible})
+	err = ra.validateContacts(context.Background(), &[]string{ansible})
 	test.AssertError(t, err, "Unknown scheme")
 
-	err = ra.validateContacts(context.Background(), &[]*core.AcmeURL{nil})
+	err = ra.validateContacts(context.Background(), &[]string{""})
 	test.AssertError(t, err, "Nil AcmeURL")
```

Comment:
Nit: can you update the `AssertError` here? It isn't `nil` or an `AcmeURL` anymore.